corehq/ex-submodules/phonelog/tasks.py

```diff
@@ -17,3 +23,31 @@ def purge_old_device_report_entries():
     UserErrorEntry.objects.filter(server_date__lt=max_age).delete()
     ForceCloseEntry.objects.filter(server_date__lt=max_age).delete()
     UserEntry.objects.filter(server_date__lt=max_age).delete()
+
+
+@task
+def send_device_logs_to_sumologic(domain, xform):
+    url = getattr(settings, 'SUMOLOGIC_URL', None)
+    if url and SUMOLOGIC_LOGS.enabled(xform.form_data.get('device_id'), NAMESPACE_OTHER):
```

Comment:
Could we put this outside the task to prevent it from getting fired if the toggle isn't enabled? Just thinking that it's going to add a fair but of traffic to celery for noop tasks.

Reply:
:+1: